### DISCUSSION TITLE
V13: Is the plan to remove Newtonsoft.JSON for System.Text.JSON ?

### DISCUSSION THREAD

**Discussion by warrenbuckley** [umbraco/Umbraco-CMS#15271] [Features and ideas]
One for the lovely HQ CMS team lead @bergmania 
Are you planning on removing Newtonsoft.JSON and use System.Text.JSON across the entire CMS codebase ?

As you have the `JsonNetSerializer` and associated `IJsonSerializer` that uses Newtonsoft.Json.

I would like to add new methods to this to specify the serializing settings object, so if you are using this in your own codebase, you can use different settings away from the main one used in the CMS for a lot of things and breaking stuff.

**Comment by rasmusjp** [comment]
`Newtonsoft.Json` is planned to be removed in v14, see https://github.com/umbraco/Announcements/issues/9

**Reply by warrenbuckley** [reply]
Thanks @rasmusfjord

**Reply by bergmania** [reply]
Sounds like you should just have your own IXXXJsonSerializer interface like the core also has IConfigurationEditorJsonSerializer.

I'll close the discusstion, as @rasmusjp replyed the to rest of the question
